packages/dashboard/src/components/pageStructure/leftRail.js

```diff
@@ -193,6 +198,13 @@ function LeftRail() {
                       numNewTemplates
                     )
                   : label;
+
+              const isTemplatesEnabled =
```

Comment:
Should this be renamed to `isTemplatesDisabled`? Also, could this be moved to the line right after `{PRIMARY_PATHS.map(({ Icon, ...path }) => {` to return `null` as soon as possible?